examples/bare_etiss_processor/get_metrics.py

```diff
@@ -147,8 +193,8 @@ def printSz(sz):
     print("  read-only data: " + printSz(staticSizes["rom_rodata"]))
     print("  code:           " + printSz(staticSizes["rom_code"]))
     print("  other required: " + printSz(staticSizes["rom_misc"]))
-    print("RAM usage:        " + printSz(ramsize))
+    print("RAM usage:        " + (printSz(ramsize) if trace_available else ">= " + printSz(ramsize) + " [Warning: stack and heap usage not included]"))
     print("  data:           " + printSz(staticSizes["ram_data"]))
     print("  zero-init data: " + printSz(staticSizes["ram_zdata"]))
-    print("  stack:          " + printSz(s.usage()))
-    print("  heap:           " + printSz(h.usage()))
+    print("  stack:          " + (printSz(s.usage()) if trace_available else "unknown (missing trace file)"))
+    print("  heap:           " + (printSz(h.usage()) if trace_available else "unknown (missing trace file)"))
```

Comment:
I added this to be able to run the `get_metrics.py` script without a trace file, so just getting the ROM usage metrics from the ELF file. Is it fine implemented like (if the trace file is not found) this or should I throw an error is this is the case and eventually add an command line flag `--skip-trace` to disable tracing mode?

Reply:
Also, it would probably be useful to write the parsed/traced memory stats into a CSV file instead of just printing it to the terminal, as this would get rid of the need to parse the command line output for evaluating benchmarks...